Resources/Textures/Clothing/Head/Hats/mouseears.rsi/meta.json

```
@@ -0,0 +1,31 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "created Loren"
```

Точнее укажи кто это сделал. Пример ниже
```suggestion
  "copyright": "created by Discord member Neko Dar#8948 (783449596986785832)"
```

---

Ах, да тут ещё не в той папке

---

Окей...